src/library-authoring/collections/CollectionInfo.tsx

```diff
@@ -10,13 +10,15 @@ import { Link, useMatch } from 'react-router-dom';
 import type { ContentLibrary } from '../data/api';
 import CollectionDetails from './CollectionDetails';
 import messages from './messages';
+import { ContentTagsDrawer } from '../../content-tags-drawer';
 
 interface CollectionInfoProps {
   library: ContentLibrary,
   collectionId: string,
+  collectionUsageKey: string,
 }
 
-const CollectionInfo = ({ library, collectionId }: CollectionInfoProps) => {
+const CollectionInfo = ({ library, collectionId, collectionUsageKey }: CollectionInfoProps) => {
```

Comment:
Since we can build the collection usage key from the libraryId + collectionId, and since that usage key is really only relevant to tags, it seems cleaner to just build it here instead of passing it through from all the places that use `CollectionInfo`

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1379/commits/534c5ab41b069443cb931d378f176106f38ceebc